packages/core/src/common/_color-aliases.scss

```diff
@@ -1,192 +1,6 @@
-    </tr>
-
-    <tr>
-      <td><div class="docs-color-bubble alias-icon-color"></div></td>
-      <td><code>$pt-icon-color</code></td>
-      <td>Default icon color</td>
-    </tr>
-    <tr>
-      <td><div class="docs-color-bubble alias-icon-color-hover"></div></td>
-      <td><code>$pt-icon-color-hover</code></td>
-      <td>Hovered icon color</td>
-    </tr>
-    <tr>
-      <td><div class="docs-color-bubble alias-icon-color-disabled"></div></td>
-      <td><code>$pt-icon-color-disabled</code></td>
-      <td>Disabled icon color</td>
-    </tr>
-    <tr>
-      <td><div class="docs-color-bubble alias-icon-color-selected"></div></td>
-      <td><code>$pt-icon-color-selected</code></td>
-      <td>Selected icon color</td>
-    </tr>
-    <tr>
-      <td><div class="docs-color-bubble alias-dark-icon-color"></div></td>
-      <td><code>$pt-dark-icon-color</code></td>
-      <td>Dark theme default icon color</td>
-    </tr>
-    <tr>
-      <td><div class="docs-color-bubble alias-dark-icon-color-hover"></div></td>
-      <td><code>$pt-dark-icon-color-hover</code></td>
-      <td>Dark theme hovered icon color</td>
-    </tr>
-    <tr>
-      <td><div class="docs-color-bubble alias-dark-icon-color-disabled"></div></td>
-      <td><code>$pt-dark-icon-color-disabled</code></td>
-      <td>Dark theme disabled icon color</td>
-    </tr>
-    <tr>
-      <td><div class="docs-color-bubble alias-dark-icon-color-selected"></div></td>
-      <td><code>$pt-dark-icon-color-selected</code></td>
-      <td>Dark theme selected icon color</td>
-    </tr>
-
-    <tr>
-      <td><div class="docs-color-bubble alias-divider-black"></div></td>
-      <td><code>$pt-divider-black</code></td>
-      <td>Black divider color</td>
-    </tr>
-    <tr>
-      <td><div class="docs-color-bubble alias-dark-divider-black"></div></td>
-      <td><code>$pt-dark-divider-black</code></td>
-      <td>Dark theme black divider color</td>
-    </tr>
-    <tr>
-      <td><div class="docs-color-bubble alias-dark-divider-white"></div></td>
-      <td><code>$pt-dark-divider-white</code></td>
-      <td>Dark theme white divider color</td>
-    </tr>
-
-    <tr>
-      <td><div class="docs-color-bubble alias-code-text-color"></div></td>
-      <td><code>$pt-code-text-color</code></td>
-      <td>Code text color</td>
-    </tr>
-    <tr>
-      <td><div class="docs-color-bubble alias-code-background-color"></div></td>
-      <td><code>$pt-code-background-color</code></td>
-      <td>Code background color</td>
-    </tr>
-    <tr>
-      <td><div class="docs-color-bubble alias-dark-code-text-color"></div></td>
-      <td><code>$pt-dark-code-text-color</code></td>
-      <td>Dark theme code text color</td>
-    </tr>
-    <tr>
-      <td><div class="docs-color-bubble alias-dark-code-background-color"></div></td>
-      <td><code>$pt-dark-code-background-color</code></td>
-      <td>Dark theme code background color</td>
-    </tr>
-  </tbody>
-</table>
-
-Styleguide variables.colors
```

Comment:
this was never used, and was the culprit for our massively bloated CSS files (#2241)